### ISSUE TITLE
Incorrect regex handling

### ISSUE THREAD

**Issue by qweeze** [brunato/xmlschema#65]
Hi!
There seems to be an issue with converting XSD patterns to python regexp.
I'm currently getting this:
```
XMLSchemaValidationError: failed validating '5067746900909' with XsdPatternsFacet(["[\\S' ']{1,50}"]).
```

**Comment by brunato** [comment]
It was an error on character group representation (class `XsdRegexCharGroup` of module _regex.py_). 
The fix is available with release 0.9.30.
Thanks

**Comment by qweeze** [comment]
Thanks!
